components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -723,7 +746,43 @@ export default class PluginManagement extends AdminSettings {
                     </div>
                     {pluginsContainer}
                 </SettingsGroup>
+                {overwritePluginModal}
             </div>
         );
     }
+
+    renderOverwritePluginModal = () => {
```

Comment:
0/5 and non-blocking but we tend to have the `render()` method as the last method in the component class